compute/cloud-client/src/main/java/compute/CreateInstance.java

```diff
@@ -28,78 +28,81 @@
 import com.google.cloud.compute.v1.Operation.Status;
 import com.google.cloud.compute.v1.ZoneOperationsClient;
 import java.io.IOException;
-import java.util.concurrent.TimeUnit;
 
 public class CreateInstance {
 
   public static void main(String[] args) throws IOException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample
+    // TODO(developer): Replace these variables before running the sample.
     String project = "your-project-id";
     String zone = "zone-name";
-    String machineName = "machine-name";
-    createInstance(project, zone, machineName);
+    String instanceName = "instance-name";
+    createInstance(project, zone, instanceName);
   }
 
 
-  // creates a new instance with the provided machineName in the specified project and zone
-  public static void createInstance(String project, String zone, String machineName)
+  // Create a new instance with the provided "instanceName" value in the specified project and zone.
+  public static void createInstance(String project, String zone, String instanceName)
       throws IOException, InterruptedException {
     // Below are sample values that can be replaced.
-    // machineType: should be of the format zones/zone-name/machineTypes/machine-type. Refer: https://cloud.google.com/compute/docs/machine-types
-    // sourceImage: path to the image that is to be mounted should be specified. Refer: https://cloud.google.com/compute/docs/images
-    // diskSizeGb: the storage size of the disk that will be attached to the instance
+    // machineType: machine type of the VM being created. This value uses the format zones/{zone}/machineTypes/{type_name}. For a list of machine types, see https://cloud.google.com/compute/docs/machine-types
+    // sourceImage: path to the operating system image to mount. For details about images you can mount, see https://cloud.google.com/compute/docs/images
+    // diskSizeGb: storage size of the boot disk to attach to the instance.
+    // networkName: network interface to associate with the instance.
     String machineType = "zones/us-central1-a/machineTypes/n1-standard-1";
```

Comment:
take a look at our [guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)

These should be created using `String.format(...` and use the zone  that is passed in.